classes/Task/Rollback/RestoreFiles.php

```diff
@@ -41,11 +41,15 @@ class RestoreFiles extends AbstractTask
 {
     const TASK_TYPE = 'rollback';
 
+    const BASE_PROGRESS = 33;
```

Comment:
I wonder if we could put all the constants related to the progress bar in the same class, this could be simpler to maintain, and would allow us to more easily have an overview of the values ​​of each step